source/extensions/priv/server/ntds_jet.h

```diff
@@ -0,0 +1,76 @@
+#ifndef _METERPRETER_SOURCE_EXTENSION_PRIV_PRIV_SERVER_NTDS_JET_H
+#define _METERPRETER_SOURCE_EXTENSION_PRIV_PRIV_SERVER_NTDS_JET_H
+#include <esent.h>
+#pragma comment(lib, "esent")
+
+/*! @brief Typedef for the jetState struct. */
+typedef struct {

```

Comment:
First preference for structs is to just call them a struct, eg. 'struct jetstate'.
If you must use a typedef, make it clear it's a type, e.g. jetstate_t.